apps/macOS/FluentUIReactNativeTester_macOS.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,394 @@
+			};
+			buildConfigurationList = 9F8DD78C242C033500C5877B /* Build configuration list for PBXProject "FluentUIReactNativeTester_macOS" */;
+			compatibilityVersion = "Xcode 9.3";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 9F8DD788242C033500C5877B;
+			productRefGroup = 9F8DD792242C033500C5877B /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				9F8DD790242C033500C5877B /* FluentUIReactNativeTester_macOS */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		9F8DD78F242C033500C5877B /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				9F8DD797242C033700C5877B /* Assets.xcassets in Resources */,
+				9F8DD79A242C033700C5877B /* MainMenu.xib in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXShellScriptBuildPhase section */
+		3466E8E23A2F9343C22D1D1F /* [CP] Check Pods Manifest.lock */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
+				"${PODS_ROOT}/Manifest.lock",
+			);
+			name = "[CP] Check Pods Manifest.lock";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+				"$(DERIVED_FILE_DIR)/Pods-FluentUIReactNativeTester_macOS-checkManifestLockResult.txt",
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
+			showEnvVarsInLog = 0;
+		};
+/* End PBXShellScriptBuildPhase section */
+
+/* Begin PBXSourcesBuildPhase section */
+		9F8DD78D242C033500C5877B /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				9F8DD795242C033600C5877B /* AppDelegate.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXVariantGroup section */
+		9F8DD798242C033700C5877B /* MainMenu.xib */ = {
+			isa = PBXVariantGroup;
+			children = (
+				9F8DD799242C033700C5877B /* Base */,
+			);
+			name = MainMenu.xib;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		9F8DD79D242C033700C5877B /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 9F8DD7A3242C0FFE00C5877B /* FluentUIReactNative_debug.xcconfig */;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Don't know how to reactivate a comment thread, but can we move these to an xcconfig now that you've created one. Build settings really need specific review/intention, but that's difficult to do when it's duplicated so many places and clearly not intentional yet